src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -1,6 +1,88 @@
-import { Typography } from '@mui/material';
-import React from 'react';
+// React Imports
+import React, { useState, useEffect } from 'react';
+// MUI Imports
+import { TextField, Button } from '@mui/material';
```

Comment:
bloated mui import, unfortunately should be `import X from '@mui/material/X';`

Reply:
Addressing issue in PR #552